### PR TITLE
[Don't merge yet] Charge sessions fix groups

### PR DESCRIPTION
**Pull request by lionsdwarf**
This fix allows iteration over all group appointment attendees, not just the first group appointment's attendees.

### CODE REVIEW

File: apps/api/jobs/charge-sessions/network.js [hunk 1]
```diff
@@ -254,23 +254,21 @@ export const getApptsWithBillables = async (from, to) => {
...
+      groupAppts.map(async groupAppt => {
+        const { attendees } = await cliniko.find({
+          url: groupAppt.attendees.links.self
+        })
+        return attendees.map(attendee => ({
           ...groupAppt,
           ...toApptAttendance(attendee.arrived),
           patient: attendee.patient
-        }
+        }))
```
Comment (coltonTB) [inline apps/api/jobs/charge-sessions/network.js]: Is there a way to lower the complexity of this logic a bit? Also not seeing the necessity of using `_.flatten`

Comment (lionsdwarf) [reply apps/api/jobs/charge-sessions/network.js]: This function is creating an Appointment record per Attendee (of which there can be multiple) on the Group Appointment. Without `_.flatten` the function will return an array of arrays.

I'm not really seeing how to simplify the logic here, but open to suggestions.

File: apps/api/test/jobs/charge-sessions.test.js [hunk 2]
```diff
@@ -822,3 +830,86 @@ test('failureNotice doesnt return for old failed charges', () => {
   }
   expect(failureNotice(rollup)).toEqual(null)
 })
+
+describe('formatGroupApptsAsIndividualAppts', () => {
```
Comment (craigspaeth) [inline apps/api/test/jobs/charge-sessions.test.js]: Sweet! Thank you for the extensive tests.

**Comment by craigspaeth** [comment]
This LGTM! Let's hold off on merging this until we pair on it together testing against prod data with charging off.

**Comment by craigspaeth** [comment]
Closing for now b/c we have some other billing changes we want to roll out carefully first. Will re-open next week. Awesome work Joe 🙌

**Pull request closed without merging**